source/connection-monitoring-and-pooling/tests/logging/connection-logging.yml

```diff
@@ -192,5 +216,3 @@ tests:
               message: "Connection checkout failed"
               serverHost: { $$type: string }
               serverPort: { $$type: [int, long] }
-              reason: "An error occurred while trying to establish a new connection"
-              error: { $$exists: true }
```

Comment:
+1

Reply:
Somehow I missed this change when I originally reviewed the PR. What was the reasoning?

Reply:
@BorisDog did you have some issue with this?

Reply:
No issues.
I thought the reason was missing, and was added now.